src/routes/RemediationDetailsComponents/DetailsCard.js

```diff
@@ -44,11 +44,11 @@ const DetailsCard = ({
   refetchAllRemediations,
 }) => {
   const [editing, setEditing] = useState(false);
-  const [value, setValue] = useState(details?.name);
+  const [value, setValue] = useState(details?.name || '');
```

Comment:
Why?

Reply:
Just curious... define "safer" 😀...

The thing is that, safer can mean that it doesn't throw an error if the `value` is `undefined`, however, that also makes it very difficult to identify issues where it should be `undefined`. I would avoid doing this and let `name` be `undefined` as long as it is not explicitly set to a string value.

It's honestly also not a good practice to throw `?.` everywhere for the same reason.

Errors are not a bad thing, not handling them properly is. 😊